models/account.py

```diff
@@ -0,0 +1,142 @@
+from dataclasses import dataclass
+from typing import List
+
+from config.env import env
+from models.application import Application
+from models.application import ApplicationMethods
+from models.data import get_data
+from models.data import post_data
+from models.magic_link import MagicLinkMethods
+from models.notification import Notification
+
+
+@dataclass
+class Account(object):
+    id: str
+    email: str
+    applications: List[Application]
+
+    @staticmethod
+    def from_json(data: dict):
+        return Account(
+            id=data.get("account_id"),
+            email=data.get("email_address"),
+            applications=data.get("applications"),
+        )
+
+
```

Comment:
💘 Fantastic type hinting in this file - I see a future in statically typed languages for you.